utilities/business-licence-upload/upload-business-licences.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo -e "-------- Getting access_token --------\n"
+TOKEN=$(curl -X POST "${KEYCLOAK_URL}/realms/${KEYCLOAK_REALM}/protocol/openid-connect/token" \
+  -H "Content-Type: application/x-www-form-urlencoded" \
+  -d "grant_type=client_credentials" \
+  -d "client_id=${KEYCLOAK_CLIENT_ID}" \
+  -d "client_secret=${KEYCLOAK_CLIENT_SECRET}" | python -c "import sys, json; print(json.load(sys.stdin)['access_token'])")
+echo -e ${TOKEN}
+
+FILE=$1
+
+echo -e "-------- Uploading ${FILE} --------\n"
+DOC_GUID=$(curl -X POST --data-binary "@${FILE}" -H "Authorization: Bearer ${TOKEN}" "${DOCMAN_DOCUMENTS_URL}?folder=licences&filename=${FILE}" | python -c "import sys, json; print(json.load(sys.stdin)['document_guid'])")
```

Comment:
Currently, Business License Documents are uploaded to the folder "business_licences"

![image](https://user-images.githubusercontent.com/39168456/120251971-01dab680-c238-11eb-98b0-ab1e6332b15e.png)

This should probably upload to the same folder. If we would rather save them in a new folder it should be named more explicitly like "paper_form_business_licenses" or "manually_uploaded_business_licenses" or something